### ISSUE TITLE
PESCAn and sequencing depth

### ISSUE THREAD

**Issue by odovgusha** [robinweide/GENOVA#295]
**Describe the bug**

Hi, I am doing aggregate analysis for several regions in two samples (wt/exp). There is a clear difference in the enrichment. Unfortunately, the samples have different sequencing depth and I think it might have impacted the results. Thus, I am asking whether the samples are normalized for sequencing depth in the PESCAn function internally or if one should normalize those beforehand. If yes, which tool would you suggest?

**Screenshots**
![image](https://user-images.githubusercontent.com/28574913/169772188-b3b4253b-8601-4b00-8175-826fdf0ce062.png)
**Desktop (please complete the following information):**

**Comment by teunbrand** [comment]
Yes the samples are, by default, compensated for sequencing depth in the `load_contacts()` step, through the `scale_bp` argument. It simply scales the values such that they sum to a billion (or whatever `scale_bp` you provide).

If you want to be sure that your result is robust against various normalisations, you might try setting `scale_cis = TRUE` or `z_norm = TRUE` and repeat your analysis.
